Bottom: 91f74e793030d72df8ee815d8d38d1b52e5df348
Top:    a2d76ec6bcf98685b7cc16f948fbd57ca25ef2d7
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2008-11-03 22:15:30 -0500

Refresh of jds-lirc

---

diff --git a/drivers/input/ir/Kconfig b/drivers/input/ir/Kconfig
index 29657d0..b854900 100644
--- a/drivers/input/ir/Kconfig
+++ b/drivers/input/ir/Kconfig
@@ -2,7 +2,8 @@
 # LIRC driver(s) configuration
 #
 menuconfig INPUT_IR
-	bool "Infrared Remote (IR) receiver/transmitter drivers"
+	select CONFIGFS_FS
+	tristate "Infrared Remote (IR) receiver/transmitter drivers"
 	default n
 	help
 	  Say Y here, and all supported Infrared Remote Control IR
